harness/determined/common/experimental/trial.py

```diff
@@ -18,6 +18,37 @@ def _to_bindings(self) -> bindings.v1LogLevel:
         return bindings.v1LogLevel(self.value)
 
 
+_csb = bindings.v1GetTrialCheckpointsRequestSortBy
+
+
+class CheckpointSortBy(enum.Enum):
+    """
+    Specifies the field to sort a list of checkpoints on.
+    """
+
+    UNSPECIFIED = _csb.SORT_BY_UNSPECIFIED.value
```

Comment:
I see that my `TrialSortBy` introduces an UNSPECIFIED, but probably it shouldn't have.

Can you just omit this from the enum?  UNSPECIFIED is like an ugly leakage from the protobuf implementation, it shouldn't appear in the user api